### PR TITLE
publish croutonjs to npm on tag

### PR DESCRIPTION
**Pull request by pjaudiomv**
closes https://github.com/bmlt-enabled/crouton/issues/259

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,6 +30,15 @@ jobs:
     - mkdir -p $DIST_DIR_S3 && cp $BUILD_DIR/$ZIP_FILENAME $DIST_DIR_S3/$ZIP_FILENAME
     - mkdir -p $DIST_DIR_GITHUB && cp $BUILD_DIR/$ZIP_FILENAME $DIST_DIR_GITHUB/$GITHUB_RELEASE_FILENAME
     deploy:
+      - provider: npm
+        email: pjaudiomv@gmail.com
```
Comment (radius314) [inline .travis.yml:34]: maybe we should use the service account?

Comment (radius314) [reply .travis.yml:34]: can we invite fortyseven like we do for wordpress plugins?

Comment (pjaudiomv) [reply .travis.yml:34]: yea wasnt sure. whats that email

Comment (pjaudiomv) [reply .travis.yml:34]: ah yes ok< I gotcha

Comment (pjaudiomv) [reply .travis.yml:34]: I think you have to invite him, use admin bmlt email

**Pull request merged**